oracle/src/utils/utils.js

```diff
@@ -107,7 +107,8 @@ const invert = p => new Promise((res, rej) => p.then(rej, res))
 const promiseAny = ps => invert(Promise.all(ps.map(invert)))
 
 const readAccessLists = {}
-async function readAccessListFile(fileName) {
+async function readAccessListFile(fileName, logger) {
+  logger.debug({ fileName }, 'Access list file read requested')
```

Comment:
I would either move this line to be within the "if" statement or, if the filename was not configured, report about this.